src/dom/attributes/attribute-manager.js

```diff
@@ -9,6 +9,55 @@
  *
  * The Attribute Manager is specific to adding custom attributes and describing their implementation (handlers). (The [Dom Manager](../../) contains the general implementation.)
  *
+ *
+ * **Examples**
+ *
+ * Built-in attribute managers like `data-f-value` and `data-f-foreach` automatically bind variables in your project's model to particular HTML elements. However, the value of the variable itself isn't always what you want in the UI.
+ *
+ * One application of custom attribute managers is to display particular HTML elements or content based on the value of a model variable, without displaying the value of the variable itself. For example, when your model variable meets a certain condition, you may want to show specific text in your UI based on the value of that variable. Creating a custom attribute manager is an easy way to do this:
```

Comment:
"content based on the value of a model variable, without displaying the value of the variable itself." is slightly confusing. Maybe just say "instead of displaying the value directly"?